setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup
+
+
+setup(
+    name='psfws',
+    version='0.1',
+    description='generate realistic wind and turbulence parameters for atmospheric point-spread function simulations',
+    url='https://github.com/LSSTDESC/psf-weather-station',
+    author='Claire-Alice Hebert',
+    author_email='chebert@stanford.edu',
+    license='MIT',
+    packages=['psfws'],
+    package_data={'psfws': ['data/*.pkl', 'data/*.p', 'data/*.npy']},
```

Comment:
You might look at how we do this in imSim (similar in GalSim too), which I think is a bit neater.  We have a function `all_files_from`, which makes a list of all files from a particular directory (and subdirectories in case that ever becomes useful).  So you could write `data_files = all_files_from('data')`, and then here `package_data={'psfws': data_files}`.